src/main/java/app/controller/CommentController.java

```diff
@@ -29,6 +30,7 @@
 public class CommentController {
     private final CommentService commentService;
 
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
As I know, you don't need to add ROLE_

Reply:
One day I've spent a lot of time trying to fix 401 status while the problem was ROLE_ absence =) Let it be)